### ISSUE TITLE
Docker version - no menu for Lightning even if enabled in docker-compose.yml

### ISSUE THREAD

**Issue by gorrdy** [mempool/mempool#4186]
### Description

I do have these settings in docker-compose.yml file

LIGHTNING_ENABLED: "true"
LIGHTNING_BACKEND: "lnd"
LIGHTNING_TOPOLOGY_FOLDER: ""
LIGHTNING_STATS_REFRESH_INTERVAL: 600
LIGHTNING_GRAPH_REFRESH_INTERVAL: 600
LIGHTNING_LOGGER_UPDATE_INTERVAL: 30

The connection to LND works as expected and data is being populated. I can access them at /lightning URL but I cannot get the Lightning button in the top menu.

I expect the docker is missing mempool-frontend-config.json and the "LIGTHNING: true" equivalent in docker-compose.yml. Is there any docu how to add the mempool-frontend-config.json parameters to the docker-compose.yml file as specified for backend?

#### Version

9f007d78f8b096365543745e7f19c721913fd62d

### Steps to reproduce

Run mempool, you cannot see the lightning in the top menu

### Expected behaviour

I'd expect there is the Lightning in the top menu when LIGHTNING_ENABLED in docker-compose.yml is set to "true".

### Actual behaviour

The Lightning is missing in the top menu

### Screenshots

<img width="1424" alt="Snímek obrazovky 2023-08-19 v 8 21 13" src="https://github.com/mempool/mempool/assets/80678558/c3e5104b-e982-44ae-9a75-875acaa04639">

#### Device or machine

ubuntu, intel nuc x86

**Comment by softsimon** [comment]
The Lightning tab is hidden until `LIGHTNING: true` in the frontend configuration as well.

**Comment by gorrdy** [comment]
Ok, but there isn't a documented way how to do that in Docker. 
https://github.com/mempool/mempool/tree/master/docker

**Comment by nymkappa** [comment]
Yeah for some reason we've never documented the frontend docker variables. PR welcome 😉. In the mean time you can find all variable here https://github.com/mempool/mempool/blob/master/docker/frontend/entrypoint.sh.

You can override those by setting them in https://github.com/mempool/mempool/blob/master/docker/docker-compose.yml in the `services.web.environment` section.

So to display the lightning menu, try to set this in your docker-compose.yml and restart.

```
services:
  web:
    environment:
        LIGHTNING: true
```

**Comment by raelschiffler** [comment]
I know this is not the right place to ask for that, but would you guys mind teaching me how to display the lightning menu in a self-hosted deploy not using docker? (I'm running mempool project built with npm)

**Comment by nymkappa** [comment]
> I know this is not the right place to ask for that, but would you guys mind teaching me how to display the lightning menu in a self-hosted deploy not using docker? (I'm running mempool project built with npm)

In your `mempool-frontend-config.json` set `LIGHTNING: true`.
